railties/test/generators/shared_generator_tests.rb

```diff
@@ -28,9 +28,9 @@ def test_skeleton_is_created
 
   def assert_generates_with_bundler(options = {})
     generator([destination_root], options)
-    generator.expects(:bundle_command).with('install').once
-    generator.stubs(:bundle_command).with('exec spring binstub --all')
```

Comment:
@kaspth in some cases `bundle_command` is called only one time `with` 'install' and some points it called two times `with` 'install' and 'exec spring binstub --all'. In `Mocha` it's stubbed only if called with 'exec spring binstub --all'.  Can you please help how to handle it?